friends/src/App.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import { Route, Link } from 'react-router-dom';
+
+import './App.css';
+import Header from './componenet/Header';
+import Footer from './componenet/Footer';
+import FriendList from './componenet/FriendList';
+
+
+
+
+
+class App extends Component {
+  constructor(){
+    super();
+    this.state={
+      friends: [{
```

Comment:
Good job setting up this initial state! It may make more sense to just set friends to an empty array, but I like this approach